src/Raven.Server/Documents/Queries/ShardedQueryResult.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.CompareExchange;
+using Raven.Client.Documents.Operations.Counters;
+using Raven.Client.Documents.Operations.TimeSeries;
+using Raven.Server.Documents.Includes;
+using Raven.Server.Documents.Queries.Explanation;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Queries;
```

Comment:
do not put sharded items under same namespace as regular ones. Create .Sharding folder here

Reply:
done